prov/gni/include/gnix_ep.h

```diff
@@ -59,7 +59,9 @@ enum {
 	GNIX_SMSG_T_RNDZV_START,
 	GNIX_SMSG_T_RNDZV_FIN,
 	GNIX_SMSG_T_RMA_DATA,
-	GNIX_SMSG_T_AMO_CNTR
+	GNIX_SMSG_T_AMO_CNTR,
+	GNIX_SMSG_T_RNDZV_IOV_START,
+	GNIX_SMSG_T_RMA_IOV_DATA
```

Comment:
> Did you see a reason why the GNIX_SMSG_T_RMA_DATA implementation wouldn't work for the IOV case?

No, this was added with the expectation that we would need a separate iov callback.  If we change the header format for the rma control message to send an iov count as well we should be able to use the existing GNIX_SMSG_T_RMA_DATA implementation.